internal/pkg/application/decoder/decoder.qalcosonic.go

```diff
@@ -21,7 +21,7 @@ func Qalcosonic_Auto(ctx context.Context, ue application.SensorEvent, fn func(co
 
 	buf := bytes.NewReader(ue.Data)
 	if buf.Len() < 42 {
-		return fmt.Errorf("w1b decoder not implemented or payload to short")
+		return fmt.Errorf("w1b decoder not implemented or payload to short (%d)", buf.Len())
```

Comment:
peculiar error message ... is the w1b decoder implemented if the payload is too (two o) short?

Reply:
👍🏼